packages/ffe-searchable-dropdown-react/less/ffe-searchable-dropdown.less

```diff
@@ -23,6 +23,10 @@
             height: auto;
             border: 1px solid var(--ffe-g-border-color);
         }
+
+        &--postListElement {
```

Comment:
CSS-klasser er jevnt over navngitt med kebab-case fremfor camelCase, så for å følge konvensjonen hadde det vært fint om denne heller kunne hete `&--post-list-element`.

Reply:
👍

Reply:
[596c707](https://github.com/SpareBank1/designsystem/pull/1579/commits/596c7079a493c0abe9659e6c1d60849f373335ca)